src/main/java/dan200/computercraft/core/apis/http/HTTPRequest.java

```diff
@@ -138,6 +138,7 @@ public void run()
 
             // Set headers
             connection.setRequestProperty( "accept-charset", "UTF-8" );
+            connection.setRequestProperty( "user-agent", "ComputerCraft/1.80" );
```

Comment:
Just need to be wary about updating the version number of ComputerCraft in the useragent if we include it there...